infra/charts/turing/requirements.yaml

```diff
@@ -14,3 +14,13 @@ dependencies:
     repository: file://subcharts/merlin
     tags:
       - mlp
+  - name: xp-management
+    version: 0.1.2
+    repository: https://turing-ml.github.io/charts
+    tags:
+      - mlp
```

Comment:
Although this is the default experiment engine for Turing, I think we should use a different tag for it, so that it can be turned off if an external user really doesn't want it (and so doesn't want to provision resources for it).

I think `xp` may be good enough as a tag and it can be set to `true` by default.

Reply:
Updated the tag to be `xp` for deploying XP experiment engine with Turing.